cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -106,14 +106,14 @@ func livenessProbe(host string, port int, scheme v1.URIScheme) *v1.Probe {
 // CreateStaticPodFiles creates all the requested static pod files.
 func CreateStaticPodFiles(manifestDir string, cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.APIEndpoint, componentNames ...string) error {
 	// TODO: Move the "pkg/util/version".Version object into the internal API instead of always parsing the string
-	k8sVersion, err := version.ParseSemantic(cfg.KubernetesVersion)
+	_, err := version.ParseSemantic(cfg.KubernetesVersion)
```

Comment:
This can be safely dropped here.

Reply:
Hmm, what happens if the user passes an invalid version? (not semantic or garbage?)

Reply:
nvm, your suggestion is at `CreateStaticPodFiles`, it's fine.

Reply:
@rosti Done